cmd/utils/cmd.go

```diff
@@ -312,3 +313,28 @@ func ExportPreimages(db ethdb.Database, fn string) error {
 	log.Info("Exported preimages", "file", fn)
 	return nil
 }
+
+func ensureSufficientMemory(sigc chan os.Signal) {
+	for t := range time.Tick(5 * time.Second) {
+		select {
+		case <-sigc:
+			log.Info("Node is being terminated...")
+		default:
+			go func() {
+				var stat syscall.Statfs_t
+				wd, err := os.Getwd(); err != nil {
+					Fatalf("Error reading available memory of Node: %v", err)
```

Comment:
Please avoid `Fatalf` -- that one causes an immediate `os.Exit`, which means it will almost certainly cause data loss and/or database corruption. We only use it in cases where things are already irrevocably broken beyond repair. 
Just use `log.Warn` (and perhaps send a SIGTERM?)

Reply:
Addressed in c355ae4c3b6bd709e781d37051814f3586130d9f